scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -189,31 +197,24 @@ task CreateAndTieOutVds {
 
         export PYSPARK_SUBMIT_ARGS='--driver-memory 16g --executor-memory 16g pyspark-shell'
         pip install --upgrade pip
-        pip install hail==0.2.120
+        pip install hail==0.2.120 ## TODO this feels like something we are going to want to change?
```

Comment:
what's the best way to deal with this? Do we want this to be something that we pass in?

Reply:
(future pr?)

Reply:
Yeah if not here then please make a ticket. I don't know that it's even guaranteed that the 0.2.120 is going to be compatible with the VDS that was generated with a later version of Hail.

Reply:
Oh you did put it in GvsUtils for the other usage... this could just be an input parameter.

Reply:
I agree that you might as well take advantage of it already being part of our "set all of our versions of things in GetToolVersions" strategy.